R-package/README.md

```diff
@@ -126,6 +126,30 @@ Please visit [demo](https://github.com/microsoft/LightGBM/tree/master/R-package/
 * [Leaf (in)Stability](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/leaf_stability.R)
 * [Weight-Parameter Adjustment Relationship](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/weight_param.R)
 
+Testing
+-------
+
+The R package's unit tests are run automatically on every commit, via integrations like [Travis CI](https://travis-ci.org/microsoft/LightGBM/) and [Azure DevOps](https://dev.azure.com/lightgbm-ci/lightgbm-ci/_build). Adding new tests in `R-package/tests/testthat` is a valuable way to improve the reliability of the R package.
+
+When adding tests, you may want to use test coverage to identify untested areas and to check if the tests you've added are covering all branches of the intended code.
+
+The example below shows how to generate code coverage for the R package on a Mac or Linux setup. To adjust for your environment, swap out the 'Install' step with [the relevant code from the instructions above](#install).
```

Comment:
> The example below shows how to generate code coverage for the R package on a Mac or Linux setup.

Should be: The example below shows how to generate code coverage for the R package on a Mac or Linux setup with gcc 8.

Reply:
I think the official current name is `macOS` when we speak about the operating system and `Mac` for computer (hardware unit). Here it is used with `Linux`, so I guess `macOS` is more appropriate.

Reply:
ok good comments both, thank you! I will change to `macOS` and make it clear that I'm using `gcc-8`

Reply:
updated in https://github.com/microsoft/LightGBM/pull/2664/commits/a06fdbcec6a02e3b87be561bb793fb614fdea383